nixos/modules/system/boot/systemd.nix

```diff
@@ -830,6 +833,7 @@ in
     systemd.targets.local-fs.unitConfig.X-StopOnReconfiguration = true;
     systemd.targets.remote-fs.unitConfig.X-StopOnReconfiguration = true;
     systemd.services.systemd-binfmt.wants = [ "proc-sys-fs-binfmt_misc.automount" ];
+    systemd.services.systemd-importd.path = [ pkgs.gnutar pkgs.btrfs-progs pkgs.gnupg-minimal ];
```

Comment:
Container's image is [always btrfs](https://github.com/systemd/systemd/blob/v231/src/shared/machine-pool.c#L241), there is no choice, and it is formatted by executing ```mkfs.btrfs```.

@kragniz in #12347 offered to separate all container-related stuff (```systemd-nspawn```, ```systemd-machined```, ```machinectl``` and ```systemd-importd```) into a separate package ```systemd-container``` or ```systemd.container``` in order not to include them (and they dependencies) in install which do not need containers.